doc/source/index.rst

```diff
@@ -60,3 +60,34 @@ Index
    msg
    run_engine
    api_changes
+
+.. toctree::
+   :hidden:
+   :caption: Data Collection
+
+   bluesky <https://nsls-ii.github.io/bluesky>
+   ophyd <https://nsls-ii.github.io/ophyd>
+
+.. toctree::
+   :hidden:
+   :caption: Data Access and Management
+
+   databroker <https://nsls-ii.github.io/databroker>
+   amostra <https://nsls-ii.github.io/amostra>
+   datamuxer <https://nsls-ii.github.io/datamuxer>
+   suitcase <https://nsls-ii.github.io/suitcase>
+
+.. toctree::
+   :hidden:
+   :caption: Data Analysis
+
+   scikit-beam <http://scikit-beam.github.io/scikit-beam>
```

Comment:
Should we include this one? It sort of breaks the "domain-agnostic" vibe that we're pushing in the data acquisition stack. Not sure how important that is.

Reply:
scikit-beam has a different style, indicating it is outside of the standard stack, just link to it from the top page (http://nsls-ii.github.io/)

Reply:
Agree. Will keep it in the docs repo, but will remove from the project-specific documentation.